plugin/delimitMate.vim

```diff
@@ -368,6 +368,11 @@ augroup delimitMate
 				\   call <SID>DelimitMateDo() |
 				\   let b:delimitMate_was_here = 1 |
 				\ endif
+
+	" Keep track of visualmode(), which is used in s:get_left_motion(),
+	" to skip leaving insert mode in block-wise visual mode.
+	autocmd InsertEnter * let b:delimitMate_visualmode = visualmode()
```

Comment:
If insert mode is entered from visual mode, `mode()` returns `n` for some reason. So we can't use that...